code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -75,7 +75,7 @@
 	. = ..()
 	if(!is_wizard_or_friend(user))
 		to_chat(user, span_hypnophrase("<span style='font-size: 24px'>The staff feels weaker as you touch it</span>"))
```

Comment:
why do we have both :(

Reply:
i dont know man but it's a pretty cool chat color